dashboard/web/pages/settings.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <v-card>
+    <v-card-title class="headline">{{$t('settings.TITLE')}}</v-card-title>
+    <div class="divider"></div>
+    <v-card-text>
+      <v-layout wrap>
+        <v-flex xs12>
+          <v-subheader class="pa-0">{{$t('settings.TITLE')}}</v-subheader>
+          <NodeRunStatus :node-status="this.nodeStatus"></NodeRunStatus>
+          <!--<v-btn color="primary" :loading="restarting"
+                 :disabled="restarting" @click="restarting = !restarting">{{$t('RESTART')}}
+          </v-btn>-->
+        </v-flex>
+        <v-flex xs12 md6>
+          <v-subheader class="pa-0">{{$t('settings.BENEFICIARY_TITLE')}}</v-subheader>
+          <material-notification class="mb-3" color="warning" v-model="beneficiaryWaringAlert">
+            {{$t('settings.BENEFICIARY_WARING', { beneficiaryAddr: beneficiaryAddr})}}
```

Comment:
If we update beneficiary address as I mentioned in comments, there is no need to restart nknd to make it effective